pages/common/pngquant.md

```diff
@@ -0,0 +1,36 @@
+# pngquant
+
+> PNG converter and lossy image compressor.
+> More information: <https://pngquant.org/>.
+
+- Compress a PNG as much as possible and write result to a new file:
+
+`pngquant {{path/to/file.png}}`
+
+- Compress a PNG and override original:
+
+`pngquant --ext .png -f {{path/to/file.png}}`
+
+- Try to compress a PNG with custom quality (skip if below the min value):
```

Comment:
There are multiple fixes available: please chose the most appropriate one.

```suggestion
- Try to compress a specific PNG with custom quality (skip if below the min value):
```

Optional change: `a specific` may sound better.

```suggestion
- Try to compress a specific .png with custom quality (skip if below the min value):
```

Optional change: you can use `.ext` extension format instead of `EXT` one.